opensquirrel/ir.py

```diff
@@ -270,13 +265,7 @@ def _normalize_axis(axis: NDArray[np.float64]) -> NDArray[np.float64]:
         """
         return axis / np.linalg.norm(axis)
 
-    @overload
-    def __getitem__(self, i: int, /) -> np.float64: ...
-
-    @overload
-    def __getitem__(self, s: slice, /) -> list[np.float64]: ...
-
-    def __getitem__(self, index: int | slice, /) -> np.float64 | list[np.float64]:
+    def __getitem__(self, index: int, /) -> np.float64:  # type:ignore[override]
```

Comment:
I agreed to this change. Why are we ignoring this?

Reply:
That's a question for @GuyPuts. This went in as a change in CQT-115. If you consider it, we can revert it.

Reply:
I added it to his PR (with his permission), because he was having difficulties with mypy.

Reply:
So you want to revert this `type:ignore[override]` change?

Reply:
Yes, please. That is, if it doesn't fail 😅

Reply:
It looks it doesn-t.